apps/dashboard/src/lib/components/Course/components/Ceritficate/certificates/BlueProfessionalBadge.svelte

```diff
@@ -0,0 +1,53 @@
+<script>
+  import { course } from '$lib/components/Course/store';
+  import { currentOrg } from '$lib/utils/store/org';
+
+  export let studentName = '';
+  let logo = '/logo-512.png';
+
+  let dateIssued = new Date().toLocaleDateString();
```

Comment:
`let` issue here, please also go to the other files to update.

Also is there a reason this is within a `certificates` folder?

Please these files should be on the same level with `Design.svelte`